src/navigation/_DevToolbarButtons.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import MenuButton from '../menu-button/_MenuButton';
+
+const propTypes = {
+  /**
+   * Current Theme
+   */
+  selectedTheme: PropTypes.string,
+
+  /**
+   * Current Locale
+   */
+  selectedLocale: PropTypes.string,
+
+  /**
+   * config for themes and locales
+   */
+  config: PropTypes.shape({
+    themes: PropTypes.arrayOf(PropTypes.string),
+    selectedTheme: PropTypes.string,
+    locales: PropTypes.arrayOf(PropTypes.string),
+    selectedLocale: PropTypes.string,
+    directions: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Will the directions ever not be `['ltr', 'rtl']`? It seems like it's hard coded in the settings config output: https://github.com/cerner/terra-dev-site/pull/175/files#diff-4f928fa6c0bb0b0946d1c3dc8a5b01adR16